pkg/printer/printer_test.go

```diff
@@ -1206,9 +1206,9 @@ Connected Nodes: 2/2`,
 			args:    args{ss},
 			wantErr: false,
 			expected: `
-          NUM FLOWS: 2031
-          MAX FLOWS: 4095
-         SEEN FLOWS: 2348885
+          NUM FLOWS: 2,031
+          MAX FLOWS: 4,095
+         SEEN FLOWS: 2,348,885
           UPTIME NS: 301515181665
```

Comment:
do we not want it here either?

Reply:
Yeah, I was wondering about this one. We could even format this as a readable duration no? Something like `1h20` etc.

Reply:
Pushed a commit to implement this, PTAL.